include/magma/vk/swapChain.h

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include "physicalDevice.h"
+#include "magma/vk/physicalDevice/PhysicalDevice.h"
```

Comment:
Move down this line please

Reply:
This part will be refactored with the SwapChain moving to hpp